components/EventForm/index.js

```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+import { useRouter } from "next/router";
+
+export default function EventForm({
+  title,
+  date,
+  type,
+  description,
+  addEvent,
+}) {
+  const router = useRouter();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const newEvent = Object.fromEntries(formData);
+    addEvent(newEvent);
+    router.push("/");
+  }
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <label htmlFor="title">Title:</label>
+          <input type="text" id="title" name="title" required></input>
```

Comment:
```suggestion
          <label htmlFor="title">Title: 
          <input type="text" id="title" name="title" required></input></label>
```

Reply:
You should nest the inputs inside the label like in the suggestion. Please do the same with the other inputs.

Reply:
@Roland-Hufnagel there's two ways to do it like you can see from this  [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). I prefer doing it like this. Thanks for the suggestion.